### PR TITLE
Quaternion: Avoid cyclic dependency on Vector3

### PR DESCRIPTION
**Pull request by WestLangley**
The inline comments should make it clear how, and why, this was done.

If for some reason, cyclic dependencies are preferred, this can easily be reverted.

### CODE REVIEW

**Comment by WestLangley** [comment]
I forgot about #14466. :-)

Anyway, this PR, plus #16021 and #15996 should eliminate the cyclic dependencies in r.102.

File: src/math/Quaternion.js [hunk 1]
```diff
@@ -354,15 +353,12 @@ Object.assign( Quaternion.prototype, {
 
 		// assumes direction vectors vFrom and vTo are normalized
 
-		var v1 = new Vector3();
```
Comment (Mugen87) [inline src/math/Quaternion.js]: If you remove `v1` here, the closure is not necessary anymore. It has no benefit for primitive values.

Comment (WestLangley) [reply src/math/Quaternion.js]: Good point. Let's see if removing the dependencies are approved. If so, I will make your requested change. Thanks for reviewing.

**Comment by mrdoob** [comment]
Thanks!

File: src/math/Quaternion.js (hunk 1 above)
Comment (WestLangley) [reply src/math/Quaternion.js]: #16039

**Pull request merged**